mteb/models/promptriever_models.py

```diff
@@ -11,12 +11,10 @@
 
 from .repllama_models import RepLLaMAWrapper
 
-logging.basicConfig(level=logging.WARNING)
 logger = logging.getLogger(__name__)
 
-EncodeTypes = Literal["query", "passage"]
-
 
+# todo PromptrieverWrapper(RepLLaMAWrapper
 class PromptrieverWrapper(RepLLaMAWrapper):
```

Comment:
@orionw Models in this file should use `RepLLaMAWrapper` or `PromptrieverWrapper`?

Reply:
The Promptriever models can re-use a lot of code from RepLlama, so they inherit from them. But they are a different model.  

Does this answer the question? Sorry if I misunderstood

Reply:
In this file, you created the `PromptrieverWrapper` class, but the models are using the `RepLLaMAWrapper` class

Reply:
Ah nice catch, thank you!  It looks like the differences are minimal (just making sure the query ends with a punctuation) so we’re probably fine but I can also submit a separate PR to make sure nothing breaks (since I apparently haven’t run it).

Thanks again for pointing it out!